dts/bindings/sensor/ti,ina7xx.yaml

```
@@ -0,0 +1,159 @@
+# Copyright The Zephyr Project Contributors
+# SPDX-License-Identifier: Apache-2.0
+
+description: Texas Instruments Bidirectional Current/Power Sensor
+
+compatible: "ti,ina7xx"
+
+include: [sensor-device.yaml, i2c-device.yaml]
+
+properties:
+  inatype:
+    type: int
+    required: true
+    description: |
+      select type of INA7xx device.
+      1 = INA700
+      2 = INA745/746
+      3 = INA780/781
+
+  convdly:
+    type: int
+    required: true
+    description: |
+      Sets the Delay for initial ADC conversion in steps of 2 ms.
+      0h = 0 s
+      1h = 2 ms
+      FFh = 510 ms
+
+  mode:
+    type: int
+    default: 0xf
+    description: |
+      MODE for continuous or triggered mode onbus voltage.#
+      0h = Shutdown
+      1h = Triggered bus voltage, single shot
+      4h = Triggered temperature, single shot
+      5h = Triggered temperature and bus voltage, single shot
+      6h = Triggered temperature and current, single shot
+      7h = Triggered temperature, current and bus voltage, single shot
+      8h = Shutdown
+      9h = Continuous bus voltage only
+      Ch = Continuous temperature only
+      Dh = Continuous bus voltage and temperature
+      Eh = Continuous temperature and current
+      Fh = Continuous temperature, current, and bus voltage
+    enum:
+      - 0x0
+      - 0x1
+      - 0x4
+      - 0x5
+      - 0x6
+      - 0x7
+      - 0x8
+      - 0x9
+      - 0xc
+      - 0xd
+      - 0xe
+      - 0xf
+
+  vbusct:
+    type: int
+    default: 5
```

```suggestion
    default: 0
```
I think this should be set to reset default

---

no, 5 is better, it's the power on reset value.